### PR TITLE
OSDOCS-3735: Clarify ROSA and OSD process security documentation

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.10, 4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3735

Link to docs preview:
ROSA: 
http://file.rdu.redhat.com/bmcelvee/20221406/rosa_policy_service_definition/rosa-policy-process-security.html#rosa-policy-change-management_rosa-policy-process-security

OSD: 
http://file.rdu.redhat.com/bmcelvee/20221406/osd_policy/policy-process-security.html#policy-change-management_policy-process-security
(VPN required) 

Requested approvals: 
@billmvt 
@wgordon17 
@davemulford 
@arendej / @0kashi
@xueli181114 I'm not sure if this change requires QE review, but I wanted to loop you in just in case

FYI @AndrewJones-RH

### CODE REVIEW

**Comment by davemulford** [comment]
LGTM. Much easier to understand!
/label px-approved

**Comment by billmvt** [comment]
LGTM, thanks!

File: modules/policy-change-management.adoc [hunk 1]
```diff
@@ -6,51 +6,32 @@
 [id="policy-change-management_{context}"]
```
Comment (bmcelvee) [inline modules/policy-change-management.adoc:6]: To help with review, this is the OSD update.

File: modules/rosa-policy-change-management.adoc [hunk 2]
```diff
@@ -7,53 +7,33 @@
 = Change management
```
Comment (bmcelvee) [inline modules/rosa-policy-change-management.adoc:7]: To help with review, this is the ROSA update.

File: modules/rosa-policy-change-management.adoc [hunk 3]
```diff
@@ -7,53 +7,33 @@
...
-SRE only uses manual changes as a fallback process because manual intervention is considered to be a failure of change management.
-====
+[id="rosa-policy-customer-initiated-chaneges_{context}"]
+== Customer-initiated changes
+
+You can initiate changes using self-service capabilities such as cluster deployment, worker node scaling, or cluster deletion.
 
-[id="rosa-policy-configuration-management_{context}"]
-== Configuration management
+Change history is captured in the *Cluster History* section in OpenShift Cluster Manager *Overview tab*, and is available for you to view. The change history includes, but is not limited to, logs from the following changes:
```
Comment (jeana-redhat) [inline modules/rosa-policy-change-management.adoc:17]:
```suggestion
Change history is captured in the *Cluster History* section in the OpenShift Cluster Manager *Overview tab*, and is available for you to view. The change history includes, but is not limited to, logs from the following changes:
```

File: modules/rosa-policy-change-management.adoc [hunk 4]
```diff
@@ -7,53 +7,33 @@
...
 
-The infrastructure and configuration of the {product-title} environment is managed as code. SRE manages changes to the {product-title} environment using a GitOps workflow and automated CI/CD pipeline.
+* Adding or removing identity providers
+* Adding or removing users to or from the dedicated-admins group
```
Comment (jeana-redhat) [inline modules/rosa-policy-change-management.adoc:20]: I think people will understand but maybe:

```suggestion
* Adding users to or removing users from the dedicated-admins group
```

Also should `dedicated-admins` be marked up in any way? (I have no idea)

File: modules/rosa-policy-change-management.adoc [hunk 5]
```diff
@@ -7,53 +7,33 @@
...
+* Scaling the cluster compute nodes
+* Scaling the cluster load balancer
+* Scaling the cluster persistent storage
+* Upgrading the cluster
 
-Each proposed change undergoes a series of automated verifications immediately upon check-in. Changes are then deployed to a staging environment where they undergo automated integration testing. Finally, changes are deployed to the production environment. Each step is fully automated.
+[id="rosa-policy-red-hat-initiated-changes_{context}"]
+== Red Hat-initiated changes
+
+Red Hat site reliability engineering (SRE) manages the infrastructure, code, and configuration of {product-title} using a GitOps workflow and fully automated CI/CD pipelines. This process ensures Red Hat can safely introduce service improvements on a continuous basis without negatively impacting customers.
```
Comment (jeana-redhat) [inline modules/rosa-policy-change-management.adoc:29]:
```suggestion
Red Hat site reliability engineering (SRE) manages the infrastructure, code, and configuration of {product-title} using a GitOps workflow and fully automated CI/CD pipelines. This process ensures that Red Hat can safely introduce service improvements on a continuous basis without negatively impacting customers.
```

File: modules/policy-change-management.adoc [hunk 6]
```diff
@@ -6,51 +6,32 @@
...
 = Change management
 
+This section describes the policies about how cluster and configuration changes, patches, and releases are managed.
 
-Cluster changes are initiated in one of two ways:
+[id="policy-customer-initiated-chaneges_{context}"]
```
Comment (jeana-redhat) [inline modules/policy-change-management.adoc:11]:
```suggestion
[id="policy-customer-initiated-changes_{context}"]
```

File: modules/rosa-policy-change-management.adoc [hunk 7]
```diff
@@ -7,53 +7,33 @@
...
-* Feedback on the change is collected and analyzed.
-* Potential gaps are diagnosed to understand resistance and automate similar change requests.
-* Corrective actions are implemented.
+This section describes the policies about how cluster and configuration changes, patches, and releases are managed.
 
-[NOTE]
-====
-SRE only uses manual changes as a fallback process because manual intervention is considered to be a failure of change management.
-====
+[id="rosa-policy-customer-initiated-chaneges_{context}"]
```
Comment (jeana-redhat) [inline modules/rosa-policy-change-management.adoc:12]:
```suggestion
[id="rosa-policy-customer-initiated-changes_{context}"]
```

File: modules/policy-change-management.adoc [hunk 8]
```diff
@@ -6,51 +6,32 @@
...
-* A customer initiates changes through self-service capabilities like cluster deployment, worker node scaling, and cluster deletion.
-* An SRE initiates a change through Operator-driven capabilities like configuration, upgrade, patching, or configuration changes.
+You can initiate changes using self-service capabilities such as cluster deployment, worker node scaling, or cluster deletion.
 
-Change history is captured in the *Cluster History* section in {cluster-manager} *Overview* tab and is available to customers. This includes logs from the following changes:
+Change history is captured in the *Cluster History* section in OpenShift Cluster Manager *Overview tab*, and is available for you to view. The change history includes, but is not limited to, logs from the following changes:
 
 * Adding or removing identity providers
-* Adding or removing users to/from the dedicated-admins group
+* Adding or removing users to or from the dedicated-admins group
```
Comment (jeana-redhat) [inline modules/policy-change-management.adoc:19]: Same comment as for ROSA

File: modules/policy-change-management.adoc [hunk 9]
```diff
@@ -6,51 +6,32 @@
...
-Cluster changes are initiated in one of two ways:
+[id="policy-customer-initiated-chaneges_{context}"]
+== Customer-initiated changes
 
-* A customer initiates changes through self-service capabilities like cluster deployment, worker node scaling, and cluster deletion.
-* An SRE initiates a change through Operator-driven capabilities like configuration, upgrade, patching, or configuration changes.
+You can initiate changes using self-service capabilities such as cluster deployment, worker node scaling, or cluster deletion.
 
-Change history is captured in the *Cluster History* section in {cluster-manager} *Overview* tab and is available to customers. This includes logs from the following changes:
+Change history is captured in the *Cluster History* section in OpenShift Cluster Manager *Overview tab*, and is available for you to view. The change history includes, but is not limited to, logs from the following changes:
```
Comment (jeana-redhat) [inline modules/policy-change-management.adoc:16]:
```suggestion
Change history is captured in the *Cluster History* section in the OpenShift Cluster Manager *Overview tab*, and is available for you to view. The change history includes, but is not limited to, logs from the following changes:
```

File: modules/policy-change-management.adoc [hunk 10]
```diff
@@ -6,51 +6,32 @@
...
-** Change request capturing change details is submitted for management approval.
-* Managing change
-** Automated nightly CI/CD jobs pick up the change and run tests.
-** The change is made to integration and stage environments, and manually validated before updating the customer cluster.
-** Major change notifications are sent before and after the event.
-* Reinforcing the change
-** Feedback on the change is collected and analyzed.
-** Potential gaps are diagnosed in order to understand resistance and automate similar change requests.
-** Corrective actions are implemented.
+Red Hat site reliability engineering (SRE) manages the infrastructure, code, and configuration of {product-title} using a GitOps workflow and fully automated CI/CD pipelines. This process ensures Red Hat can safely introduce service improvements on a continuous basis without negatively impacting customers.
```
Comment (jeana-redhat) [inline modules/policy-change-management.adoc:28]:
```suggestion
Red Hat site reliability engineering (SRE) manages the infrastructure, code, and configuration of {product-title} using a GitOps workflow and fully automated CI/CD pipelines. This process ensures that Red Hat can safely introduce service improvements on a continuous basis without negatively impacting customers.
```

**Review by jeana-redhat — COMMENTED** [review]
Just a few small things to consider here

**Comment by wgordon17** [comment]
LGTM, thanks @bmcelvee

File: modules/rosa-policy-change-management.adoc (hunk 4 above)
Comment (bmcelvee) [reply modules/rosa-policy-change-management.adoc:20]: I think so. Good catch!

**Pull request merged**